### PR TITLE
fix: icons alignment and product url

### LINKED ISSUE #11370: Icons alignment issue on /wallets/#sim page
### Describe the bug

Icons used in some lists inside `/wallets/#sim` page are not aligned properly with text

### To reproduce

1. Go to https://ethereum.org/en/wallets/#sim
2. Scroll down to **Types of wallets** and **How to stay safe** sections
3. See icons not aligned properly

### Expected behavior

Icons aligned properly with text

### Screenshots

Screen Shot 2023-10-04 at 13 20 08

Screen Shot 2023-10-04 at 13 20 41

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional context

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

### LINKED ISSUE #11390: Replace ethereum.org's product roadmap URL on /about page
### Describe the bug

Replace current ethereum.org's product roadmap URL on [/about/#roadmap page](https://ethereum.org/en/about/#roadmap) to [Q1 2024 ethereum.org product roadmap](https://github.com/ethereum/ethereum-org-website/issues/12005). 

### To reproduce

1. Go to https://ethereum.org/en/about/#roadmap
2. Replace URL and copy: '[View our 2023 Q1 product roadmap](https://github.com/ethereum/ethereum-org-website/issues/9090)' URL to ['View our 2024 Q1 product roadmap](https://github.com/ethereum/ethereum-org-website/issues/12005)'

### Expected behavior

n/a

### Screenshots

_No response_

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional context

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**
## Description
fixed icons alignment in [this](https://ethereum.org/en/wallets/#sim) page

## Related Issue
fixes #11370
fixes #11390

### CODE REVIEW

**Comment by MahendraBishnoi29** [comment]
@corwintines @pettinarip i think this PR is ready to merge please have a look.

File: src/pages-conditional/wallets.tsx [hunk 1]
```diff
@@ -331,6 +336,7 @@ const WalletsPage = ({
                 emoji={type.emoji}
                 description={type.description}
                 emojiSize={2.5}
+                alignItems="center"
```
Comment (pettinarip) [inline src/pages-conditional/wallets.tsx:339]: This should also be fixed inside of the `HorizontalCard` component.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Thanks for the PR @MahendraBishnoi29

File: src/pages-conditional/wallets.tsx (hunk 1 above)
Comment (MahendraBishnoi29) [reply src/pages-conditional/wallets.tsx:339]: thanks for these @pettinarip

**Comment by corwintines** [comment]
This seems to have been abandoned, so im going to close this PR out and leave the issue open. @MahendraBishnoi29, if you want to continue working on it, feel free to open a new PR with the changes in the HorizontalCard component :)

**Pull request closed without merging**